qiskit/providers/ibmq/api/clients/websocket.py

```diff
@@ -44,7 +44,14 @@
 # `asyncio` by design does not allow event loops to be nested. Jupyter (really
 # tornado) has its own event loop already so we need to patch it.
 # Patch asyncio to allow nested use of `loop.run_until_complete()`.
-nest_asyncio.apply()
+# Before applying the patch, check if an event loop is available otherwise
+# create one and set it active
+try:
+    LOOP = asyncio.get_event_loop()
+except RuntimeError:
+    LOOP = asyncio.new_event_loop()
+asyncio.set_event_loop(LOOP)
```

Comment:
You don't actually need to set the event loop here since it's not used (yet). `nest_asyncio.apply()` will use the loop you pass in and not the active loop.